web/src/state/Explore.tsx

```diff
@@ -61,7 +61,7 @@ export const exploreSettingsDefault: ExploreSettings = {
   loading: true,
   hexagonClicked: null,
   hexagonHighlight: null,
-  viewMode: ExploreViewMode.BOTH,
+  viewMode: ExploreViewMode.LIST,
```

Comment:
why you making this change? i don't understand. wasn't it supposed to be defined by the administration on the network configuration?

Reply:
Yes but, I'm using the list state by default until we don't have the admin variable